PWGJE/Tasks/nucleiInJets.cxx

```diff
@@ -170,14 +170,14 @@ struct nucleiInJets {
   Configurable<bool> applySkim{"applySkim", false, "Apply skimming"};
   Configurable<std::string> cfgSkim{"cfgSkim", "fHighFt0Mult", "Configurable for skimming"};
 
-  static constexpr float gMassProton = 0.93827208f;
-  static constexpr float gMassDeuteron = 1.87561f;
-  static constexpr float gMassTriton = 2.80892f;
-  static constexpr float gMassHelium = 2.80839f;
+  static constexpr float kMassProton = o2::constants::physics::MassProton;
+  static constexpr float kMassDeuteron = o2::constants::physics::MassDeuteron;
+  static constexpr float kMassTriton = o2::constants::physics::MassTriton;
+  static constexpr float kMassHelium = o2::constants::physics::MassHelium3;
   static constexpr int PDGProton = 2212;
-  static constexpr int PDGDeuteron = 1000010020;
-  static constexpr int PDGTriton = 1000010030;
-  static constexpr int PDGHelium = 1000020030;
+  static constexpr int PDGDeuteron = o2::constants::physics::Pdg::kDeuteron;
+  static constexpr int PDGTriton = o2::constants::physics::Pdg::kTriton;
+  static constexpr int PDGHelium = o2::constants::physics::Pdg::kHelium3;
```

Comment:
Use the existing PDG code names directly. Also for `kProton`.

Reply:
Indeed. I will fix this in my next PR.